pandas/core/nanops.py

```diff
@@ -829,9 +830,42 @@ def _checked_add_with_arr(arr, b):
 
     Raises
     ------
-    OverflowError if any x + y exceeds the maximum int64 value.
+    OverflowError if any x + y exceeds the maximum or minimum int64 value.
     """
-    if (np.iinfo(np.int64).max - b < arr).any():
-        raise OverflowError("Python int too large to "
-                            "convert to C long")
+    # For performance reasons, we broadcast 'b' to the new array 'b2'
+    # so that it has the same size as 'arr'.
+    if np.__version__ >= LooseVersion('1.10.0'):
```

Comment:
use the pd _version_undernp11 that are already defined (look up the spelling of the variable)